archunit/src/main/java/com/tngtech/archunit/core/domain/JavaType.java

```diff
@@ -82,7 +92,113 @@ public interface JavaType extends HasName {
      * @return All raw types involved in this {@link JavaType}
      */
     @PublicAPI(usage = ACCESS)
-    Set<JavaClass> getAllInvolvedRawTypes();
+    default Set<JavaClass> getAllInvolvedRawTypes() {
```

Comment:
The body of the message of the commit ``fix `getAllInvolvedRawTypes()` recursing infinitely`` contains the following line, which probably shouldn't be there:
> This reverts commit 6f23f66fd7f355fe40bea5b5a0fcd18a17e2e783.

(At least I don't know what 6f23f66fd7f355fe40bea5b5a0fcd18a17e2e783 should be. 😉)

Reply:
Mhmm, good catch, probably from rebasing and juggling commits 🙈